### PR TITLE
fix: typos across specs

### PR DESCRIPTION
**Pull request by nschonni**
This probably needs to get re-submitted in smaller chunks. I went across the repo using cSpell and fixed word-by-word creating this large PR.
Possible alternate approach would be to collapse this down and then re-submit by spec folder.

If going with the flattened folder-by-folder, should the various `issue*` files be ignored? There were some with incorrect key names.

### CODE REVIEW

**Comment by fantasai** [comment]
Wow, this must have taken a long time to get right. Thanks for putting it together! No need to split across specs, it's actually easier to review like this. Want to confirm s/targetted/targeting/ with @tabatkins, but aside from that everything else looks correct. The one bit that needs adjustment is that in the issues-* files, removing a letter from a key needs to add a space so that the values all continue to line up. :)

**Comment by nschonni** [comment]
I had separately pulled out the "Typogrraphy" one, and @svgeesus pointed out that some of the older specs like css-fonts-3 have hit the point were they shouldn't really be touched.
If you want to comment inline on any specific ones that should be pulled or reformmated, I can try and rebase out the particular changes

File: css-display-3/issues-wd-2017.bsi [hunk 1]
```diff
@@ -833,7 +833,7 @@ Closed:   Invalid
 Issue 98.
 Summary:  Swap where default inner/outer display types are defined
 From:     Oriol
-Commment: https://github.com/w3c/csswg-drafts/issues/2658
+Comment: https://github.com/w3c/csswg-drafts/issues/2658
```
Comment (fantasai) [inline css-display-3/issues-wd-2017.bsi:836]:
```suggestion
Comment:  https://github.com/w3c/csswg-drafts/issues/2658
```

File: css-display-3/issues-wd-2017.bsi [hunk 2]
```diff
@@ -814,7 +814,7 @@ From:     Amelia Bellamy-Royds
 Comment:  https://github.com/w3c/csswg-drafts/issues/2502
 Response: https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-378809174
 Closed:   Accepted
-Veified:  https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551
+Verified:  https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551
```
Comment (fantasai) [inline css-display-3/issues-wd-2017.bsi:817]:
```suggestion
Verified: https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551
```

File: css-display-3/issues-wd-2017.bsi [hunk 3]
```diff
@@ -187,7 +187,7 @@ Summary:  Block containers can also establish an inline formatting context
 From:     Oriol
 Comment:  https://github.com/w3c/csswg-drafts/issues/1185
 Response: https://github.com/w3c/csswg-drafts/issues/1185#issuecomment-301622704
-Cloed:    Accepted
+Closed:    Accepted
```
Comment (fantasai) [inline css-display-3/issues-wd-2017.bsi:190]:
```suggestion
Closed:   Accepted
```

File: css-display-3/issues-wd-2017.html [hunk 4]
```diff
@@ -227,7 +227,7 @@ <h1>CSS Display Module Level 3 Disposition of Comments for 2015-10-15 WD</h1>
 <span>From:     Oriol</span>
 <span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/1185'>https://github.com/w3c/csswg-drafts/issues/1185</a></span>
 <span>Response: <a href='https://github.com/w3c/csswg-drafts/issues/1185#issuecomment-301622704'>https://github.com/w3c/csswg-drafts/issues/1185#issuecomment-301622704</a></span>
-<span>Cloed:    Accepted</span>
+<span>Closed:    Accepted</span>
```
Comment (fantasai) [inline css-display-3/issues-wd-2017.html:230]:
```suggestion
<span>Closed:   Accepted</span>
```

File: css-display-3/issues-wd-2017.html [hunk 5]
```diff
@@ -854,7 +854,7 @@ <h1>CSS Display Module Level 3 Disposition of Comments for 2015-10-15 WD</h1>
 <span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/2502'>https://github.com/w3c/csswg-drafts/issues/2502</a></span>
 <span>Response: <a href='https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-378809174'>https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-378809174</a></span>
 <span class="a">Closed:   Accepted</span>
-<span>Veified:  <a href='https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551'>https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551</a></span></pre>
+<span>Verified:  <a href='https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551'>https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551</a></span></pre>
```
Comment (fantasai) [inline css-display-3/issues-wd-2017.html:857]:
```suggestion
<span>Verified: <a href='https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551'>https://github.com/w3c/csswg-drafts/issues/2502#issuecomment-399498551</a></span></pre>
```

File: css-display-3/issues-wd-2017.html [hunk 6]
```diff
@@ -873,7 +873,7 @@ <h1>CSS Display Module Level 3 Disposition of Comments for 2015-10-15 WD</h1>
 <span>Issue 98. <a href='#issue-98'>#</a></span>
 <span>Summary:  Swap where default inner/outer display types are defined</span>
 <span>From:     Oriol</span>
-<span>Commment: <a href='https://github.com/w3c/csswg-drafts/issues/2658'>https://github.com/w3c/csswg-drafts/issues/2658</a></span>
+<span>Comment: <a href='https://github.com/w3c/csswg-drafts/issues/2658'>https://github.com/w3c/csswg-drafts/issues/2658</a></span>
```
Comment (fantasai) [inline css-display-3/issues-wd-2017.html:876]:
```suggestion
<span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/2658'>https://github.com/w3c/csswg-drafts/issues/2658</a></span>
```

File: css-flexbox-1/issues-cr-20160526.bsi [hunk 7]
```diff
@@ -145,7 +145,7 @@ From:     fantasai
 Comment:  https://github.com/w3c/csswg-drafts/issues/316
 Closed:   Accepted
 Resolved: https://lists.w3.org/Archives/Public/www-style/2017Aug/0036.html
-Veified:  Reporter is editor
+Verified:  Reporter is editor
```
Comment (fantasai) [inline css-flexbox-1/issues-cr-20160526.bsi:148]:
```suggestion
Verified: Reporter is editor
```

File: css-flexbox-1/issues-cr-20160526.html [hunk 8]
```diff
@@ -183,7 +183,7 @@ <h1>CSS Flexible Box Layout Disposition of Comments for 2016-05-26 CR</h1>
 <span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/316'>https://github.com/w3c/csswg-drafts/issues/316</a></span>
 <span class="a">Closed:   Accepted</span>
 <span>Resolved: <a href='https://lists.w3.org/Archives/Public/www-style/2017Aug/0036.html'>https://lists.w3.org/Archives/Public/www-style/2017Aug/0036.html</a> <a href="https://www.w3.org/Mail/flatten/index?subject=https%3A%2F%2Flists.w3.org%2FArchives%2FPublic%2Fwww-style/2017Aug/0036.html"><abbr title="Thread">&#x2208;</abbr></a></span>
-<span>Veified:  Reporter is editor</span></pre>
+<span>Verified:  Reporter is editor</span></pre>
```
Comment (fantasai) [inline css-flexbox-1/issues-cr-20160526.html:186]:
```suggestion
<span>Verified: Reporter is editor</span></pre>
```

File: css-grid-1/issues-cr-2016.bsi [hunk 9]
```diff
@@ -291,7 +291,7 @@ Closed:   Retracted
 Issue 36.
 Summary:  Placing Against Missing Line Names Needs Example
 From:     Rachel Andrew
-Commment: https://github.com/w3c/csswg-drafts/issues/966
+Comment: https://github.com/w3c/csswg-drafts/issues/966
```
Comment (fantasai) [inline css-grid-1/issues-cr-2016.bsi:294]:
```suggestion
Comment:  https://github.com/w3c/csswg-drafts/issues/966
```

File: css-grid-1/issues-cr-2016.html [hunk 10]
```diff
@@ -331,7 +331,7 @@ <h1>CSS Grid Layout Level 1 Disposition of Comments for 2016-09-29 CR</h1>
 <span>Issue 36. <a href='#issue-36'>#</a></span>
 <span>Summary:  Placing Against Missing Line Names Needs Example</span>
 <span>From:     Rachel Andrew</span>
-<span>Commment: <a href='https://github.com/w3c/csswg-drafts/issues/966'>https://github.com/w3c/csswg-drafts/issues/966</a></span>
+<span>Comment: <a href='https://github.com/w3c/csswg-drafts/issues/966'>https://github.com/w3c/csswg-drafts/issues/966</a></span>
```
Comment (fantasai) [inline css-grid-1/issues-cr-2016.html:334]:
```suggestion
<span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/966'>https://github.com/w3c/csswg-drafts/issues/966</a></span>
```

File: css-grid-1/issues-cr-2017.bsi [hunk 11]
```diff
@@ -54,7 +54,7 @@ Summary:  Clarify axis for AMS condition
 From:     Mats Palmgren
 Comment:  https://github.com/w3c/csswg-drafts/issues/2283
 Changes:  https://github.com/w3c/csswg-drafts/commit/4b784426189a2979bf0b396a6ccf8c6b763004a8
-Respone:  https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213
+Response:  https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213
```
Comment (fantasai) [inline css-grid-1/issues-cr-2017.bsi:57]:
```suggestion
Response: https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213
```

File: css-grid-1/issues-cr-2017.bsi [hunk 12]
```diff
@@ -318,7 +318,7 @@ Issue 34.
 Summary:  Rename min-size contribution
 From:     Tab Atkins
 Comment:  https://github.com/w3c/csswg-drafts/issues/3277
-Rsponse:  https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037
+Response:  https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037
```
Comment (fantasai) [inline css-grid-1/issues-cr-2017.bsi:321]:
```suggestion
Response: https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037
```

File: css-grid-1/issues-cr-2017.bsi [hunk 13]
```diff
@@ -820,7 +820,7 @@ Comment:  https://github.com/w3c/csswg-drafts/issues/4783
 Response: https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-587749012
 Closed:   Accepted
 Resolved: CSSWG 2020-07-27
-Veified:  https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434
+Verified:  https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434
```
Comment (fantasai) [inline css-grid-1/issues-cr-2017.bsi:823]:
```suggestion
Verified: https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434
```

File: css-grid-1/issues-cr-2017.html [hunk 14]
```diff
@@ -94,7 +94,7 @@ <h1>CSS Grid Layout Level 1 Disposition of Comments for 2017-12-14 CR</h1>
 <span>From:     Mats Palmgren</span>
 <span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/2283'>https://github.com/w3c/csswg-drafts/issues/2283</a></span>
 <span>Changes:  <a href='https://github.com/w3c/csswg-drafts/commit/4b784426189a2979bf0b396a6ccf8c6b763004a8'>https://github.com/w3c/csswg-drafts/commit/4b784426189a2979bf0b396a6ccf8c6b763004a8</a></span>
-<span>Respone:  <a href='https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213'>https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213</a></span>
+<span>Response:  <a href='https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213'>https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213</a></span>
```
Comment (fantasai) [inline css-grid-1/issues-cr-2017.html:97]:
```suggestion
<span>Response: <a href='https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213'>https://github.com/w3c/csswg-drafts/issues/2283#issuecomment-363643213</a></span>
```

File: css-grid-1/issues-cr-2017.html [hunk 15]
```diff
@@ -358,7 +358,7 @@ <h1>CSS Grid Layout Level 1 Disposition of Comments for 2017-12-14 CR</h1>
 <span>Summary:  Rename min-size contribution</span>
 <span>From:     Tab Atkins</span>
 <span>Comment:  <a href='https://github.com/w3c/csswg-drafts/issues/3277'>https://github.com/w3c/csswg-drafts/issues/3277</a></span>
-<span>Rsponse:  <a href='https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037'>https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037</a></span>
+<span>Response:  <a href='https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037'>https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037</a></span>
```
Comment (fantasai) [inline css-grid-1/issues-cr-2017.html:361]:
```suggestion
<span>Response: <a href='https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037'>https://github.com/w3c/csswg-drafts/issues/3277#issuecomment-446021037</a></span>
```

File: css-grid-1/issues-cr-2017.html [hunk 16]
```diff
@@ -860,7 +860,7 @@ <h1>CSS Grid Layout Level 1 Disposition of Comments for 2017-12-14 CR</h1>
 <span>Response: <a href='https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-587749012'>https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-587749012</a></span>
 <span class="a">Closed:   Accepted</span>
 <span>Resolved: CSSWG 2020-07-27</span>
-<span>Veified:  <a href='https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434'>https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434</a></span></pre>
+<span>Verified:  <a href='https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434'>https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434</a></span></pre>
```
Comment (fantasai) [inline css-grid-1/issues-cr-2017.html:863]:
```suggestion
<span>Verified: <a href='https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434'>https://github.com/w3c/csswg-drafts/issues/4783#issuecomment-667237434</a></span></pre>
```

**Review by fantasai — APPROVED** [review]

**Comment by fantasai** [comment]
If you can merge in those whitespace fixes, I can merge in your PR.

**Review by fantasai — APPROVED** [review]
You need to unmark it as a draft for me to merge.

**Comment by nschonni** [comment]
@fantasai I'm un-marked it as draft, but I'm not sure what parts might still need to be rebased out because they've hit CR now

**Comment by fantasai** [comment]
None, it merged cleanly. :) Thanks~

**Pull request merged**